roles/lib_openshift/src/doc/clusterrole

```diff
@@ -10,7 +10,6 @@ description:
 options:
   state:
     description:
-    - Supported states, present, absent, list
```

Comment:
Lets copy `State controls the action that will be taken with resource` here as well

Reply:
Fixed! However there's a travis-problem now and I don't know the reason..

Reply:
```
Checking generated module code: roles/lib_openshift/src/generate.py
('Generated content does not match for /home/travis/build/openshift/openshift-ansible/roles/lib_openshift/src/../library/oc_adm_csr.py',)
Checking generated module code: roles/lib_utils/src/generate.py
Found errors while generating module code.
```